doc/config_rsmexplain.rst.inc

```diff
@@ -83,9 +89,26 @@ If we want to explain a random sample of the responses in explain_data, we can s
 
     Only one of ``sample_ids``, ``sample_range`` or ``sample_size`` must be specified. If none of them are specified, explanations will be generated for the entire set of responses in ``explain_data`` which could be very slow, depending on its size.
 
+.. _section_order_rsmexplain:
+
+section_order *(Optional)*
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A list containing the order in which the sections in the report should be generated. Any specified order must explicitly list:
+
+    1. Either *all* pre-defined sections if a value for the :ref:`general_sections <general_sections_rsmexplain>` field is not specified OR the sections specified using :ref:`general_sections <general_sections_rsmexplain>`, and
+
+    2. *All* custom section names specified using :ref:`custom_ sections <custom_sections_rsmexplain>`, i.e., file prefixes only, without the path and without the `.ipynb` extension, and
+
+.. _show_auto_cohorts_rsmexplain:
+
 show_auto_cohorts *(Optional)*
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-If this option is set to ``true``, auto cohort bar plots will be displayed. These plots can be useful for detecting interaction effects between cohorts and features. If a cohort shows a high feature value, then there may be an interaction between that cohort and the feature. Defaults to ``false``. These plots are not shown by default because these plots may be unstable or provide misleading information if ``explain_data`` is not large enough. For smaller datasets, SHAP may not be able to detect strong feature interactions and compute clear cohorts. If that happens, the plots wil be too specific to be useful.
+If this option is set to ``true``, auto cohort bar plots will be displayed. These plots can be useful for detecting interaction effects between cohorts and features. If a cohort shows a high feature value, then there may be an interaction between that cohort and the feature. Defaults to ``false``. These plots are not shown by default because these plots may be unstable or provide misleading information if ``explain_data`` is not large enough. For smaller datasets, SHAP may not be able to detect strong feature interactions and compute clear cohorts. If that happens, the plots wil be too specific to be useful. If you have a large enough dataset, you can set this option to ``true`` and see if the plots are useful.
```

Comment:
```suggestion
If this option is set to ``true``, auto cohort bar plots will be displayed. These plots can be useful for detecting interaction effects between cohorts and features. If a cohort shows a high feature value, then there may be an interaction between that cohort and the feature. Defaults to ``false``. These plots are not shown by default because these plots may be unstable or provide misleading information if ``explain_data`` is not large enough. For smaller datasets, SHAP may not be able to detect strong feature interactions and compute clear cohorts. If that happens, the plots will be too specific to be useful. If you have a large enough dataset, you can set this option to ``true`` and see if the plots are useful.
```